workers/unity/Assets/Playground/Scripts/Worker/IosClientWorkerConnector.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+using UnityEngine;
+#if UNITY_IOS
+using Improbable.Gdk.Mobile.Ios;
+#endif
+using Improbable.Gdk.Core;
+
+namespace Playground.Worker
+{
+    public class IosClientWorkerConnector : AbstractMobileClientWorker
+    {
+        protected override string GetHostIp()
+        {
+#if UNITY_IOS
```

Comment:
Does this need to be conditionally compiled away? Can it not be checked at runtime?

Reply:
nope, because we allow the mobile package only in editor, ios or android builds. therefore, we need to use these preprocessors to get a succeeding build